### ISSUE TITLE
Let mocks implement std::fmt::Debug

### ISSUE THREAD

**Issue by janhohenheim** [DavidDeSimone/mockitol#16]
It is very common to trait bind your trait to Debug:
```rust
pub trait Object: Debug {
    // ...
}
```
`#[mock]` fails here however, as the resulting mock does not implement `std::fmt::Debug`:
```
   | #[cfg_attr(test, mock)]
   |                  -----
   |                  |
   |                  `properties::MockObject` cannot be formatted using `{:?}`
   |                  in this macro invocation
   |
   = help: the trait `std::fmt::Debug` is not implemented for `properties::MockObject`
   = note: add `#[derive(Debug)]` or manually implement `std::fmt::Debug`
```

I propose simply adding a `#[derive(Debug)]` to the generated mock.

**Comment by RaynaYe** [comment]
same problem

**Comment by janhohenheim** [comment]
@RaynaYe  FYI our team was bothered enough by stuff like this to develop an own mocking library: [mockiato](https://blog.myelin.ch/2019/05/24/mockiato-announcement.html)

**Comment by DavidDeSimone** [comment]
Addressed in https://github.com/DavidDeSimone/mock_derive/pull/25. Will be included in next release, 0.9.0
